packages/core/test/unit/config.test.js

```diff
@@ -57,16 +57,9 @@ describe('Unit / Config Migration', () => {
       }
     }, mocked);
 
-    expect(mocked.migrate.map).toEqual([
-      ['snapshot.authorization', 'discovery.authorization'],
-      ['snapshot.requestHeaders', 'discovery.requestHeaders']
-    ]);
-
-    expect(logger.stderr).toEqual([
-      '[percy] Warning: The config option `snapshot.authorization` ' +
-        'will be removed in 1.0.0. Use `discovery.authorization` instead.',
-      '[percy] Warning: The config option `snapshot.requestHeaders` ' +
-        'will be removed in 1.0.0. Use `discovery.requestHeaders` instead.'
+    expect(mocked.migrate.deprecate).toEqual([
+      ['snapshot.authorization', { map: 'discovery.authorization', type: 'config', in: '1.0.0' }],
+      ['snapshot.requestHeaders', { map: 'discovery.requestHeaders', type: 'config', in: '1.0.0' }]
```

Comment:
Wonder if we should open an issue like "v1.0.0 deprecations" and list the various deprecations we've made/make sure to remove?

Reply:
That's a good idea.